src/graphics/colour.rs

```diff
@@ -0,0 +1,67 @@
+type ColourRGB = (u8, u8, u8);
```

Comment:
Maybe consider having this as a `struct` like
```rs
struct ColorRGB {
    r: u8;
    g: u8;
    b: u8;
}
// or
struct ColorRGB(u8, u8, u8);
```

so we can add methods to it in the future?